steamos-waydroid-installer.sh

```diff
@@ -397,7 +398,30 @@ else
 		elif [ "$Choice" == "TV" ]
 		then
 			echo Android TV chosen!
-			exit
+			echo Initializing Waydroid
+			echo -e "$current_password\n" | sudo -S waydroid init
+			echo Downloading Android TV image
+            echo -e "$current_password\n" | sudo -S curl -o ~/waydroid/images/androidtv.zip $ANDROID_TV_IMG -L
+			echo Extracting Archive
+			echo -e "$current_password\n" | sudo -S unzip -o ~/waydroid/images/androidtv -d ~/waydroid/images
+			echo -e "$current_password\n" | sudo -S rm ~/waydroid/images/androidtv.zip
+			echo Reinitializing Waydroid
+			echo -e "$current_password\n" | sudo -S waydroid init -f
+		    if zenity --question --text="Install MicroG and Aurora Store"; then
```

Comment:
this doesn't seem specific to Android TV? it's inside of the same conditional right now though

Reply:
also the `casualsnek` script [can install MicroG](https://github.com/casualsnek/waydroid_script/blob/1a2d3ad643206ad5f040e0155bb7ab86c0430365/main.py#L86)

Reply:
And the whitespace is off here -- this has mixed indentation, tabs and spaces.

the existing style would also put a new line and comment before this block

```suggestion

			# install MicroG and Aurora Store if desired
			if zenity --question --text="Install MicroG and Aurora Store"; then
```